recipes/splipy/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "splipy" %}
+{% set version = "1.5.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Splipy-{{ version }}.tar.gz
+  sha256: 6b204494fc7b4cc5c70d4ffa5db1cf20fd5326e23aa6493c743912b148b5ca22
+  path: PyPI_text.md
+  sha256: 9cd73d2a0d1eebaaf541e9b37bf69114c2644f2adc498f674e91d0f62429a7f6
```

I think the URL method should work, but I think you need to turn this into a yaml list:
```suggestion
  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Splipy-{{ version }}.tar.gz
    sha256: 6b204494fc7b4cc5c70d4ffa5db1cf20fd5326e23aa6493c743912b148b5ca22
  - path: PyPI_text.md
    sha256: 9cd73d2a0d1eebaaf541e9b37bf69114c2644f2adc498f674e91d0f62429a7f6
```

---

Thanks for your help.  The error messages from the system are pretty unhelpful, they make this look like a SHA256 mismatch instead of a syntax problem.  And this YAML syntax is completely unfamiliar to me.